### DISCUSSION TITLE
@Lisias Tweakscale not working on stock and DLC items

### DISCUSSION THREAD

**Discussion by Delta013** [TweakScale/TweakScale#284] [Support] [answered]
Having what seems to be a common issue.
The scale options icon in the bottom right is there and working(I think), however the slider to scale objects is nowhere to be found.
I only have recall and tweakscale installed and after updating, it stopped working.
Please help
[Player.zip](https://github.com/net-lisias-ksp/TweakScale/files/10526058/Player.zip)

**Comment by Lisias** [comment]
Hi!

There's something wrong with ModuleManagerWatchDog:

```
FileNotFoundException: Could not load file or assembly 'ModuleManagerWatchDog, Version=1.1.0.3, Culture=neutral, Public
KeyToken=null' or one of its dependencies.


(Filename:  Line: -1)

FileNotFoundException: Could not load file or assembly 'ModuleManagerWatchDog, Version=1.1.0.3, Culture=neutral, Public
KeyToken=null' or one of its dependencies.
```

 The `Player.log` doesn't have all the information I need, I need your `KSP.log` in order to check what's happening - there's a lot of messages that are only logged on `KSP.log`.
 
Alternatively, you can just remove `GameData/ModuleManagerWatchDog` from your rig. The thing is useful to detect borkage about Module Manager, but it's possible to live without it.

**Comment by Lisias** [comment]
Ah!! Using @Lisias in the Title doesn't works! You need to use it on the postage itself! :)

Cheers!

**Reply by Lisias** [reply]
Like this @Delta013 ! :)

**Comment by Delta013** [comment]
Ah right sorry,
Had a crack without watchdog in there, but it still didn't want to work.
[KSP.zip](https://github.com/net-lisias-ksp/TweakScale/files/10526155/KSP.zip)

**Comment by Lisias** [comment]
@Delta013 , there's nothing wrong on your log - TweakScale was loaded, the Sanity Checks was executed and nothing wrong related to TS itself was found.

```
[LOG 14:55:34.909] [TweakScale] INFO: WriteDryCost Concluded : 483 parts found ; 0 checks failed ; 0 parts with hotfixes ; 0 parts with issues overruled ; 0 Show Stoppers found; 0 Sanity Check failed; 483 unscalable parts.
```

**HOWEVER**

Tweakscale is telling me that you game rig has 483 parts, and from that 483 are unscalable parts - all of them. This is odd, this is what I got from my KSP 1.12.5 test bed (with both DLCs installed, as yours):

```
[LOG 14:06:47.349] [TweakScale] INFO: WriteDryCost Concluded : 483 parts found ; 0 checks failed ; 0 parts with hotfixes ; 0 parts with issues overruled ; 0 Show Stoppers found; 0 Sanity Check failed; 85 unscalable parts.
``` 

You see? Only 85 parts are not scalable (robotics or related, and I think a few new parts that I didn't wrote patches yet).

I think your installation is borked. I suggest you to uninstall everything (but `Squad` and `SquadExpansion`, obviously) from your `GameData` and start again, this time taking care and reading the `INSTALL.md `files when available.

Or perhaps you would wanna try Curse Installer or CKAN?

**Reply by Delta013** [reply]
Yea mate, uninstalled and deleted everything and tried again. I've used Curse and manual, yet have the same issue.
Not sure what I"m doing wrong.
[KSP.zip](https://github.com/net-lisias-ksp/TweakScale/files/10529373/KSP.zip)

**Comment by Lisias** [comment, marked as answer]
Whoops! My bad! I didn't had thought about that!

You need to install Module Manager!!! It's a component that apply patches into the parts using some tags in special tailored config files (thing starting with @, %, +, -, !, using brackets, etc).

(you are a newcomer to KSP, right?)

Install it Directly into the GameData.

You can find it on Forum [here](https://forum.kerbalspaceprogram.com/index.php?/topic/50533-18x-112x-module-manager-422-june-18th-2022-the-heatwave-edition/#comment-720814).

Cheers!
